components/badge/badge_pins.h

```diff
@@ -140,6 +140,40 @@
 #define MPR121_PIN_NUM_LEDS     10
 #define MPR121_PIN_NUM_SD_CD    11
 
+#elif defined(CONFIG_SHA_BADGE_V3_LITE)
```

Comment:
optionally prefix the '#include "badge_config_v3.h"' with a '#define HAVE_BADGE_MPR121' or '#undef HAVE_BADGE_MPR121'